tests/expected/elm/Client.elm

```diff
@@ -407,4 +412,24 @@ updateMeRequest prefix maybeTimeout withCredentials updateUser =
 
 
 
+paramsToQuery : List ( String, String ) -> List ( String, Maybe String ) -> String
```

Comment:
Could you please add a description of the function in the generated code? It would help a reader to trace the client code since it is not immediately clear what this function does from its name.